repo/build.py

```diff
@@ -22,7 +22,7 @@
 """
 
 def sign_xml(config, source_xml):
-	if not config.GPG_SIGNING_KEY:
+	if not config.GPG_SIGNING_KEY or os.getenv('NO_SIGN'):
```

Comment:
How about putting this check in the config, so `config.GPG_SIGNING_KEY` is `None` if it's set?

Reply:
Ok, done.